<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInscription</class>
 <widget class="QDialog" name="DialogInscription">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_email">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_prenom">
     <property name="text">
      <string>Prenom</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_nom"/>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_nom">
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_annee">
     <property name="text">
      <string>Annee</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_prenom"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBox_annee">
     <item>
      <property name="text">
       <string>L1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L2MATH</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L3MATH</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M1MATH</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M2MATH</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L2INFO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L3INFO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M1INFO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M2INFO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Autre</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_email"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_nom</tabstop>
  <tabstop>lineEdit_prenom</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>comboBox_annee</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogInscription</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogInscription</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
